options.go

```diff
@@ -89,25 +85,18 @@ func (o *ConnectionOptions) ToDialOptions() ([]grpc.DialOption, error) {
 }
 
 func (o *ConnectionOptions) transportCredentials() (grpc.DialOption, error) {
-	if o.NoTLS {
-		return grpc.WithTransportCredentials(insecure.NewCredentials()), nil
+	cfg := &TLSConfig{
```

Comment:
@gertd and alternative to this would be to replace `ClientCertPath`, `ClientKeyPath`, and `CACertPath` in the `Config` struct with a `TLS` field of type `TLSConfig`.
Then we'll be able to write:
```go
creds, err := cfg.TLS.ClientCredentials(...)
```

But that would be a breaking change in the `Config` struct.

Reply:
I would leave it as-is.